src/pages/Products/Products.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, React, useEffect } from "react";
+import "./products.css";
+import { ProductCard } from "../../components";
+import { Filters } from "./components";
+import axios from "axios";
+
+export function Products() {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    (async function () {
+      const res = await axios.get("/api/products");
```

Comment:
Suggestion : -
 
- You have done the same api call in the Home.jsx to get products data , doing the same api call twice is not a good practice , it affects performance , instead yu can store the products into the context , you can discuss this with TA and MA as well .

Reply:
This is just the UI component I haven't created context yet. Will move it there once I create the context

Reply:
Great